mediathread/factories.py

```diff
@@ -151,6 +151,14 @@ class CollaborationPolicyRecordFactory(factory.DjangoModelFactory):
     FACTORY_FOR = CollaborationPolicyRecord
 
 
+class AssignmentItemFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = AssignmentItem
```

Comment:
Heads up that `FACTORY_FOR` is deprecated in factoryboy now, so we should be switching to

```
class Meta:
    model = AssignmentItem
```

style instead.